### ISSUE TITLE
bun install doesn't work in Angular project 

### ISSUE THREAD

**Issue by ZahiriNatZuke** [oven-sh/bun#9817]
### What version of Bun is running?

1.1.0

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

1. place terminal into angular project
2. run the command `bun install` or `bun install --sasve`

### What is the expected behavior?

Install all dependencies according to package.json file

### What do you see instead?

![image](https://github.com/oven-sh/bun/assets/41224027/a96812c3-a333-46b5-8450-cbb98eba0d3b)
Don't move forward from this step, you stay there stopped.

### Additional information

Before trying bun in my angular project I used pnpm as the node package manager

**Comment by max06** [comment]
Same problem for a nuxt project with workspaces. 

Edit: Same bun version, same docker image, two different machines - one works, one is stuck.

Edit2: Removing the lockfile helped. 

Edit3: It seems the increased timeout (to 5 minutes) is an issue:

```text
 => [builder 5/7] RUN bun install  --frozen-lockfile --verbose                                                                                                           
 => => # Request: GET /typescript/-/typescript-5.4.3.tgz                                                                                                                       
 => => # Connection: keep-alive                                                                                                                                                
 => => # User-Agent: Bun/1.1.0                                                                                                                                                 
 => => # Accept: */*                                                                                                                                                           
 => => # Host: registry.npmjs.org                                                                                                                                              
 => => # Accept-Encoding: gzip, deflate, br                                                                                                                                    
```

**Comment by Stephan-MC** [comment]
have you tried setting ```bun``` as the default packageManager for angular 

```bash
ng config --global cli.packageManager bun
```


or (for current project only)
```bash
ng config cli.packageManager bun
```

**Comment by Jarred-Sumner** [comment]
This was very likely fixed in #11206. If you still run into this after upgrading to Bun v1.1.12+ or later, please let us know and I'll re-open it.
